taggit/locale/ru/LC_MESSAGES/django.po

```diff
@@ -17,6 +17,9 @@ msgstr ""
 "Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
 "X-Poedit-Language: Russian\n"
 
+msgid "Taggit"
+msgstr "Метки"
```

Comment:
In line 30 specified `msgid "Tags"`, but here is `msgid "Taggit"` that define in [taggit/apps.py](https://github.com/PixxxeL/django-taggit/blob/custom-slugify/taggit/apps.py). And my check are show that this string is not translate. But after Im add `msgid "Taggit"` - all was fine.